src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -518,6 +518,11 @@ protected override int MakeSlot(BoundExpression node)
                     }
                 case BoundKind.Conversion:
                     {
+                        int slot = getPlaceholderSlot(node);
+                        if (slot > 0)
```

Comment:
should this be `>= 0`? (I'm not sure if it's possible to observe)

Reply:
The slot is never 0. This matches other tests in this class that are either `> 0` or `< 0`.

---
In reply to: [267981582](https://github.com/dotnet/roslyn/pull/34303#discussion_r267981582) [](ancestors = 267981582)